controls/SV-204392.rb

```diff
@@ -1,54 +1,55 @@
-    # ls -la <filename>
-    -rw-------. 1 root root 133 Jan 11 13:25 /etc/audisp/audisp-remote.conf
-    If the file is more permissive than the default permissions, this is a finding.
-    If the file is not owned by the default owner and is not documented with the Information System Security Officer
-    (ISSO), this is a finding.
-    If the file is not a member of the default group and is not documented with the Information System Security Officer
-    (ISSO), this is a finding.'
-  desc  'fix', "
-    Run the following command to determine which package owns the file:
-
-    # rpm -qf <filename>
-
-    Reset the user and group ownership of files within a package with the
-following command:
-
-    #rpm --setugids <packagename>
-
-
-    Reset the permissions of files within a package with the following command:
-
-    #rpm --setperms <packagename>
-  "
+# encoding: UTF-8
+
+control "SV-204392" do
+  title "The Red Hat Enterprise Linux operating system must be configured so that the file permissions, ownership, and group membership of system files and commands match the vendor values."
+  desc "Discretionary access control is weakened if a user or group has access permissions to system files and directories greater than the default."
+  desc "default", "Discretionary access control is weakened if a user or group has access permissions to system files and
+    directories greater than the default."
+  desc "check", "Verify the file permissions, ownership, and group membership of system files and commands match the vendor values.
+
+Check the default file permissions, ownership, and group membership of system files and commands with the following command:
+
+# for i in `rpm -Va | egrep '^.{1}M|^.{5}U|^.{6}G' | cut -d \" \" -f 4,5`;do for j in `rpm -qf $i`;do rpm -ql $j --dump | cut -d \" \" -f 1,5,6,7 | grep $i;done;done
+
+/var/log/gdm 040755 root root
+/etc/audisp/audisp-remote.conf 0100640 root root
+/usr/bin/passwd 0104755 root root
+
+For each file returned, verify the current permissions, ownership, and group membership:
+# ls -la <filename>
+
+-rw-------. 1 root root 133 Jan 11 13:25 /etc/audisp/audisp-remote.conf
+
+If the file is more permissive than the default permissions, this is a finding.
+
+If the file is not owned by the default owner and is not documented with the Information System Security Officer (ISSO), this is a finding.
+
+If the file is not a member of the default group and is not documented with the Information System Security Officer (ISSO), this is a finding."
+  desc "fix", "Run the following command to determine which package owns the file:
+
+# rpm -qf <filename>
+
+Reset the user and group ownership of files within a package with the following command:
+
+# rpm --setugids <packagename>
+
+
+Reset the permissions of files within a package with the following command:
+
+# rpm --setperms <packagename>"
   impact 0.7
-  tag 'legacy': ['V-71849', 'SV-86473']
-  tag 'severity': 'high'
```

Comment:
Looks like it changed these tags from `'name'` to just `name`. Is this the standard we want to go with? Is this from rubocop or delta?

Reply:
Before running delta:
```
tag 'legacy': ['V-71849', 'SV-86473']
tag 'severity': 'high'
```

After running delta: 
```
tag legacy: ["V-71849", "SV-86473"]
tag severity: "high"
```

Looks like it's from delta. Suggestion- could have a linter (rubicon?) add the single-quotes back in OR have delta keep them.